<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TICSummary Sochi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutCentral">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxModeShow"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxId">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPrev">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Prev</string>
        </property>
        <property name="shortcut">
         <string>Left</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>500</number>
        </property>
        <property name="autoRepeatInterval">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="shortcut">
         <string>Right</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>500</number>
        </property>
        <property name="autoRepeatInterval">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionEditConnection"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionResetWindows"/>
    <addaction name="actionControl"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Calibration</string>
    </property>
    <addaction name="actionDetectorCoeffiecient"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCalibration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenLocalFolder">
   <property name="text">
    <string>Open local folder</string>
   </property>
  </action>
  <action name="actionEditConnection">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionResetWindows">
   <property name="text">
    <string>Reset positions windows</string>
   </property>
  </action>
  <action name="actionDetectorCoeffiecient">
   <property name="text">
    <string>Galo</string>
   </property>
  </action>
  <action name="actionControl">
   <property name="text">
    <string>Control</string>
   </property>
  </action>
  <action name="actionClearMonitoring">
   <property name="text">
    <string>Clear Monitoring</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
